pkg/sink/codec/decoder.go

```diff
@@ -31,7 +31,9 @@ type RowEventDecoder interface {
 	// NextResolvedEvent returns the next resolved event if exists
 	NextResolvedEvent() (uint64, error)
 	// NextRowChangedEvent returns the next row changed event if exists
-	NextRowChangedEvent() (*model.RowChangedEvent, error)
+	// if the event only has handle key sent, return true,
+	// consumer should query the row data by the handle key and commit-ts
```

Comment:
Please add some comments to explain the differences in behavior betwee protocols.